docs/ossm/ambient/observability.md

```diff
@@ -0,0 +1,300 @@
+
+# Getting Started with Observability in Service Mesh Ambient mode
+
+Red Hat OpenShift Observability provides real-time visibility, monitoring, and analysis of various system metrics, logs, and events to help you quickly diagnose and troubleshoot issues before they impact systems or applications.
+
+Red Hat OpenShift Service Mesh connects open-source observability tools and technologies to create a unified Observability solution. The components of this Observability stack work together to help you collect, store, deliver, analyze, and visualize data.
+
+The following components in Service Mesh Ambient mode generate detailed telemetry for all service communications within a mesh.
+
+| Component        | Description   |
+| -----------      | ------------- |
+| ztunnel          | generates L4 telemetry such as TCP metrics.                                           |
+| waypoint proxies | generates L7 telemetry for HTTP, HTTP/2, gRPC traffic metrics and distributed traces. |
+
+Red Hat OpenShift Service Mesh integrates with the following Red Hat OpenShift Observability components in ambient mode:
+
+- OpenShift Cluster Monitoring components
+- Red Hat OpenShift Distributed tracing platform
+
+OpenShift Service Mesh also integrates with:
+
+- Kiali provided by Red Hat, a powerful console for visualizing and managing your service mesh.
+- OpenShift Service Mesh Console (OSSMC) plugin, an OpenShift Container Platform console plugin that seamlessly integrates Kiali console features into your OpenShift console.
+
+## Configuring Metrics with OpenShift Cluster Monitoring and Service Mesh Ambient mode
+
+Monitoring stack components are deployed by default in every OpenShift Container Platform installation. These components include Prometheus, Alertmanager, Thanos Querier, and others.
+
+When you have enrolled your application in the Service Mesh Ambient mesh mode, you can monitor the [Istio Standard Metrics](https://istio.io/latest/docs/reference/config/metrics/) of your application from the ztunnel resource or the waypoint proxies. The ztunnel also exposes [a variety of DNS and debugging metrics](https://github.com/istio/ztunnel/blob/master/README.md#metrics).
```

Comment:
Is it Zunnel OR Waypoint? I thought from the [Kiali graph](https://kiali.io/docs/features/ambient/#ambient-telemetry) it should be possible to have both, right?

I think it would be worth having a sentence here to explain that because ambient mode has two level of proxy, there are also two sets of metrics that can be collected for each service. Layer 4 metrics, such as TCP, can be collected from ztunnel, while layer 7 metrics, such as HTTP can be collected from waypoint. You can collect metrics from one or both levels of proxy.

Then below, we should have a separate procedure (including separate pre-requisites) for collecting ztunnel and waypoint metrics.